src/amo/components/SearchSort/index.js

```diff
@@ -0,0 +1,79 @@
+import classNames from 'classnames';
+import React, { PropTypes } from 'react';
+import { compose } from 'redux';
+
+import SearchSortLink from 'amo/components/SearchSort/SearchSortLink';
+import translate from 'core/i18n/translate';
+
+import './SearchSort.scss';
+
+
+const DEFAULT_SORT = 'relevance';
+
+export class SearchSortBase extends React.Component {
+  static propTypes = {
+    filters: PropTypes.object.isRequired,
+    i18n: PropTypes.object.isRequired,
+    pathname: PropTypes.string.isRequired,
+  }
+
+  constructor(props) {
+    super(props);
+    this.state = { sortVisible: false };
+  }
+
+  onClick = (event) => {
+    event.preventDefault();
+    this.toggleSort();
+  }
+
+  sortOptions() {
+    const { i18n } = this.props;
+
+    return [
+      { sort: 'updated', text: i18n.gettext('Recently Updated') },
+      { sort: 'relevance', text: i18n.gettext('Relevance') },
+      { sort: 'users', text: i18n.gettext('Most Users') },
+      { sort: 'rating', text: i18n.gettext('Top Rated') },
+    ];
+  }
+
+  toggleSort() {
+    // Blur the search sort toggle if we're closing it.
+    if (this.state.sortVisible) {
+      this.searchToggle.blur();
```

Comment:
Yeah. If you're *closing* the search sort it is a lot visually nicer to blur it. I thought it would make sense for keyboard users too, but maybe it should only be `onClick` and not `onKeyDown` so it doesn't reset a keyboard user's tab position (which I assume it will do, but have to admit am not sure about).